web/src/_variables.scss

```diff
@@ -13,12 +13,14 @@ $colors-border-hover: var(--calculated_secondary-border-1);
 $colors-main-border: var(--calculated_important-border);
 $footer-border-color: var(--calculated_footer_border);
 $colors-super-pale-black: var(--calculated_super-pale_label-text-graphics);
+$colors-blue-link: var(--blue_link_dark);
```

Comment:
If the already defined blue color is not useful here and we want to define a new color anyway, it should also be added to the color utils and the dark and light color js files.

utils/color.js
config/colors/light.js
config/colors/dark.js